### PR TITLE
fix: Remove Morphic specific code

### LINKED ISSUE #2: Remove Morphic specific code
## Describe the bug

Required for the related [GPII-4345](https://issues.gpii.net/browse/GPII-4345), we need to remove all of the Morphic specific code from UIO+. Any Morphic required features will be added ontop of UIO+ in [UIO+ for Morphic](https://github.com/GPII/gpii-chrome-extension).

### Expected 

For example, things like the web socket connection should be removed from UIO+, README updated, possibly the namespace used in the code base should be updated, and etc.

### PR DESCRIPTION
**Pull request by jobara**
Removes the Morphic, APCP, and GPII specific parts from the code base. This includes changing namespaces, updating docs, and removing portions of the code base that will be included in [UIO+ for Morphic](https://github.com/GPII/gpii-chrome-extension) instead.

Resolves #2 

Includes changes from: #5, #6 and #9

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Styling classes must be prefixed with `uioPlus-`. Styling classes for components coming from Infusion may be prefixed
+with `fl-`.
+
+### Classes used by JavaScript
+
+Classes used by JavaScript to locate and manipulate DOM elements must be prefixed with `uioPlusJS-`. Classes used by
+Infusion components may be prefixed with `flc-`
+
+## Publishing to the Chrome Web Store
```
Comment (amb26) [inline CONTRIBUTING.md:72]: This should be updated with master to remove the extra diff here

File: package.json [hunk 2]
```diff
@@ -16,16 +16,15 @@
...
         "ui options",
-        "user interface options",
-        "gpii",
-        "morphic"
+        "user interface options"
     ],
     "author": "Fluid Project",
     "bugs": {
-        "url": "http://issues.gpii.net/browse/GPII"
+        "url": "https://github.com/fluid-project/uio-plus/issues"
```
Comment (amb26) [inline package.json:25]: OK, ignore my earlier comments, I see now that this work is part of this pull

**Comment by amb26** [comment]
This pull looks ok to me

**Comment by jobara** [comment]
@amb26 this PR is ready for review.

**Comment by amb26** [comment]
This looks good, I think we just need a dependency update before we publish

**Pull request merged**